docs/REST API/rest-api.md

```diff
@@ -8714,6 +8718,8 @@ Get the results of a software installation.
 
 ### List software
 
+> The new keys/values added in the app management feature are experimental and may change. You can find the upcoming breaking changes [here](https://github.com/fleetdm/fleet/pull/19291/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64L8749-R8791).
```

Comment:
```suggestion
> **This feature is still experimental.** 
>
> The new keys/values added in the app management feature are experimental and may change. You can find the upcoming breaking changes [here](https://github.com/fleetdm/fleet/pull/19291/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64L8749-R8791).
```

Reply:
I think I see what you're after but I think the purple call out box is loud enough?

I added bold to "experimental"

Not sure how the purple callout box would render with a bold title/header...